packages/@o3r/workspace/schematics/library/rules/rules.nx.ts

```diff
@@ -118,12 +119,28 @@ export function nxGenerateModule(options: NgGenerateModuleSchema & { packageJson
       renameTemplateFiles(),
       move(targetPath)
     ]);
+    const templateJest = apply(url('./templates/jest'), [
+      template({
+        ...options,
+        tsconfigBasePath: findConfigFileRelativePath(tree, ['tsconfig.base.json', 'tsconfig.json'], targetPath)
+      }),
+      renameTemplateFiles(),
+      move(targetPath)
+    ]);
     rules.push(mergeWith(templateNx, MergeStrategy.Overwrite));
+    const packageJsonContent = tree.readText('/package.json');
+    const hasJestInstalled = options.testingFramework === 'jest' || packageJsonContent.match('jest');
```

Comment:
```suggestion
    const hasJestInstalled = options.testingFramework === 'jest' || packageJsonContent.match('"jest"');
```

To allow people to create a `majesty` script :D